docs_user/modules/proc_reusing-existing-subnet-ranges.adoc

```diff
@@ -1,25 +1,14 @@
 [id="reusing-existing-subnet-ranges_{context}"]
 
-= Scenario 2: Reusing existing subnet ranges
+= Reusing existing subnet ranges
 
-This scenario is only applicable when the existing subnet ranges have enough IP
-addresses for the new control plane services. On the other hand, it allows to
-avoid additional routing configuration between the old and new subnets, as in xref:using-new-subnet-ranges_{context}[Scenario 1: Using new subnet ranges].
+You can reuse existing subnet ranges if they have enough IP addresses to allocate to the new control plane services. You configure the new control plane services to use the same subnet as you used in the adopted environment, and configure the allocation pools that are used by the new services to exclude IP addresses that are already allocated to existing cluster nodes. By reusing existing subnets, you avoid additional routing configuration between the old and new subnets. as in xref:using-new-subnet-ranges_{context}[Configuring new subnet ranges].
```

Comment:
Shouldn’t "adopted environment" say the "RHOSP environment" or the " previous environment"?